File: Assignments/2_Operators and Expression/Javascript/Solutions/Q8.js
function swap(a, b) {
    console.log(`Before swap: A = ${a}, B = ${b}`);
    a = a + b;
    b = a - b;
    a = a - b;
    console.log(`After swap: A = ${a}, B = ${b}`);
}

let a = 5;
let b = 10;
swap(a, b);

File: Assignments/2_Operators and Expression/C#/Q8.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Enter first integer (A): ");
        int a = int.Parse(Console.ReadLine());
        Console.Write("Enter second integer (B): ");
        int b = int.Parse(Console.ReadLine());

        Console.WriteLine($"Before swap: A = {a}, B = {b}");
        a = a + b;
        b = a - b;
        a = a - b;
        Console.WriteLine($"After swap: A = {a}, B = {b}");
    }
}
